.test-infra/jenkins/CommonJobProperties.groovy

```diff
@@ -34,7 +34,10 @@ class CommonJobProperties {
       String defaultBranch = 'master',
       int defaultTimeout = 100,
       boolean allowRemotePoll = true,
-      String jenkinsExecutorLabel =  'beam') {
+      String jenkinsExecutorLabel = 'beam',
+      boolean cleanWorkspace = true) {
+
+    context.disabled()
```

Comment:
For my education, what does this do?

Reply:
🙀  good point @tvalentyn . It is unwanted leftover from testing it on another Jenkins with some additional changes in CommonJobProperties. I deleted in recent fixup.

Reply:
For testing purposes on testing Jenkins jobs were disabled.
It refers to this method:
https://jenkinsci.github.io/job-dsl-plugin/#method/javaposse.jobdsl.dsl.jobs.WorkflowJob.disabled